node/errors.js

```diff
@@ -445,34 +444,71 @@ module.exports.classify = function classify(err) {
         case 'tchannel.max-pending-for-service':
             return 'Declined';
 
-        case 'tchannel.timeout':
+        case 'tchannel.socket-local-closed':
+        case 'tchannel.local.reset':
+            return 'Cancelled';
```

Comment:
I don't think that word means what you think it means, this is no a "graceful" shutdown.  A graceful shutdown would wait for all pending requests to be finished before closing / shutting down, while at the same time not accepting any more incoming requests.